admin-ui/plugins/fido/components/types/StaticConfiguration.ts

```diff
@@ -0,0 +1,102 @@
+import { FormikProps } from 'formik'
+import { AppConfiguration, FormData } from '../../types'
+
+export interface StaticConfigurationFormValues {
+  authenticatorCertsFolder: string
+  mdsCertsFolder: string
+  mdsTocsFolder: string
+  checkU2fAttestations: boolean
+  unfinishedRequestExpiration: string | number
+  metadataRefreshInterval: string | number
+  serverMetadataFolder: string
+  userAutoEnrollment: boolean
+  enabledFidoAlgorithms: string[]
+  requestedParties: RequestedPartyForm[]
+}
+
+export interface RequestedPartyForm {
+  name?: string
+  domains?: string[]
+}
+
+export interface CredentialTypeOption {
+  key: string
+  value: string
+}
+
+export interface RequestedPartyOption {
+  key: string
+  value: string
+}
+
+export interface StaticConfigurationProps {
+  fidoConfiguration: {
+    fido: AppConfiguration
+  }
+  handleSubmit: (values: FormData) => void
+}
+
+// Formik types
+export type StaticConfigurationFormik = FormikProps<StaticConfigurationFormValues>
+
+// GluuInputRow component props
+export interface GluuInputRowProps {
+  label: string
+  name: string
+  type?: string
+  value: string | number
+  formik: StaticConfigurationFormik
+  lsize: number
+  rsize: number
+  showError: boolean
+  errorMessage: string | undefined
+}
+
+// GluuToggleRow component props
+export interface GluuToggleRowProps {
+  label: string
+  name: string
+  formik: StaticConfigurationFormik
+  lsize: number
+  rsize: number
+  doc_category: string
+}
+
+// GluuProperties component props
+export interface GluuPropertiesProps {
+  compName: string
+  isInputLables: boolean
```

Comment:
Rename `isInputLables` to `isInputLabels` across the codebase  
Update the prop definition and all its references in these files:  
- admin-ui/plugins/fido/components/types/StaticConfiguration.ts  
- admin-ui/plugins/fido/components/StaticConfiguration.tsx (lines 193, 209)  
- admin-ui/plugins/fido/components/DynamicConfiguration.tsx (line 245)  
- admin-ui/app/routes/Apps/Gluu/GluuProperties.tsx (lines 19, 128)  
- admin-ui/app/routes/Apps/Gluu/GluuPropertyItem.tsx (lines 14, 29, 40)  

```diff
-  isInputLables: boolean
+  isInputLabels: boolean
```